DCF_Demo.md

```diff
@@ -556,6 +595,9 @@ az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ip
 </details>
 <br>
 
+## Safe package update
+
+One of the advantages to use DCF is the safe package update mechanism. It allows developers and administrators to test new packages on the devices before switch to this package, and rollback for the old package, if it is necessary. The tutorial [How to safe update a DCF package](DCF_Safe_Update.md) will take you through this process, using 2 versions of the key_vault package created on this tutorial. 
```

Comment:
>One of the advantages to use DCF is the safe package update mechanism. It allows developers and administrators to test new packages on the devices before switch to this package, and rollback for the old package, if it is necessary. The tutorial [How to safe update a DCF package](DCF_Safe_Update.md) will take you through this process, using 2 versions of the key_vault package created on this tutorial. [](http://example.com/codeflow?start=0&length=403)

nit: "One of the advantages of using DCF is the safe package update mechanism. It allows developers and administrators to test new packages on the devices before officially switching, It also supports rollback to the old package if necessary. The tutorial [How to safe update a DCF package](DCF_Safe_Update.md) will take you through this process using 2 versions of the key_vault package created in this tutorial."

Reply:
#Pending